src/test/java/com/todoist/pojo/PersonTest.kt

```diff
@@ -1,24 +1,24 @@
 package com.todoist.pojo
 
-import kotlin.test.assertEquals
 import kotlin.test.Test
+import kotlin.test.assertEquals
 
 class PersonTest {
     @Test
     fun getDefaultAvatarText() {
         listOf(
-            "First Last" to "FL",
-            "Not capitalized" to "NC",
-            " White\n\nSpace  " to "WS",
-            "ã ž" to "ÃŽ",
-            "Ludwig van Beethoven" to "LB",
-            "[Spe-cial], (Chars)" to "SC",
-            "\u00A0NO-BREAK\u00A0SPACE\u00A0" to "NS",
-            "Mononym" to "M",
-            "Q" to "Q",
-            "  " to "?"
-        ).forEach { (fullName, defaultAvatarText) ->
-            assertEquals(defaultAvatarText, Person.getDefaultAvatarText(fullName))
+            Triple("First Last", "test@doist.com", "FL"),
+            Triple("Not capitalized", "test@doist.com", "NC"),
+            Triple(" White\n\nSpace  ", "test@doist.com", "WS"),
+            Triple("ã ž", "test@doist.com", "ÃŽ"),
+            Triple("Ludwig van Beethoven", "test@doist.com", "LB"),
+            Triple("[Spe-cial], (Chars)", "test@doist.com", "SC"),
+            Triple("\u00A0NO-BREAK\u00A0SPACE\u00A0", "test@doist.com", "NS"),
+            Triple("Mononym", "test@doist.com", "M"),
+            Triple("Q", "test@doist.com", "Q"),
+            Triple("  ", "test@doist.com", "?"),
```

Comment:
Shouldn't this one be `T` instead of `?`?

Reply:
Yeah...just fixed it :)